### PR TITLE
Added how to uninstall service brokers

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: 
* http://file.rdu.redhat.com/~ahoffer/2019/uninstalling-service-brokers/applications/service_brokers/uninstalling-template-service-broker.html
* http://file.rdu.redhat.com/~ahoffer/2019/uninstalling-service-brokers/applications/service_brokers/uninstalling-ansible-service-broker.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@johnkim76 Could you please review what I have here so far for uninstalling the ASB and TSB?

Running through the steps, it seems to work for the TSB (all its service classes are removed), but does not work for the ASB (all its service classes are still shown in the dev catalog).

**Comment by bergerhoffer** [comment]
@openshift/team-documentation For peer review please.

File: applications/service_brokers/installing-template-service-broker.adoc [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 :context: sb-installing-tsb
 
 toc::[]
 
-You can install the template service broker to gain access to the template
+You can install the {tsb-name} to gain access to the template
 applications that it provides.
 
-IMPORTANT: The template service broker is deprecated in this release. Equivalent and better functionality is present in the Operator Framework and
+IMPORTANT: The {tsb-name} is deprecated in this release. Equivalent and better functionality is present in the Operator Framework and
```
Comment (ahardin-rh) [inline applications/service_brokers/installing-template-service-broker.adoc]: instead of "this release", should we say {product-title} 4.1?

Comment (ahardin-rh) [reply applications/service_brokers/installing-template-service-broker.adoc]: I would structure the IMPORTANT admonition as instructed by the [docs guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions). I see a few instances that can be updated in this PR.

File: modules/sb-install-service-catalog.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@
 [id="sb-install-service-catalog-{context}"]
 = Installing service catalog
 
-You must install the service catalog by completing the following steps if you plan on using any of the services from the {asb-name} or template service broker.
+You must install the service catalog by completing the following steps if you plan on using any of the services from the {asb-name} or {tsb-name}.
```
Comment (ahardin-rh) [inline modules/sb-install-service-catalog.adoc]: If you plan on using any of the services from the {asb-name} or {tsb-name}, you must install the service catalog by completing the following steps.

File: modules/sb-uninstall-asb.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* The {asb-name} is installed.
+
+.Procedure
+
+This procedure assumes that you installed the {asb-name} into the
+`ansible-service-broker` project.
+
+. Uninstall the {asb-name}.
+.. Navigate to *Catalog* -> *Installed Operators* and select the
+*ansible-service-broker* project from the drop down.
```
Comment (ahardin-rh) [inline modules/sb-uninstall-asb.adoc]: s/drop down/drop-down menu  (or just menu)

File: modules/sb-uninstall-asb.adoc [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+// TODO: Change to *OpenShift Ansible Service Broker Operator* by GA (4 of 5)
+.. Click *Automation Broker Operator*.
+.. Select the *Automation Broker* tab.
+.. Click *ansible-service-broker*.
+.. From the *Actions* drop down, select *Delete Automation Broker*.
+.. Click *Delete* from the confirmation pop up.
```
Comment (ahardin-rh) [inline modules/sb-uninstall-asb.adoc]: s/pop up/page (or window maybe?) IBM style states that we should avoid use of "pop up".

File: modules/sb-uninstall-asb.adoc [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
++
+The {asb-name} is now uninstalled, and service bundles will soon be removed
+from the Developer Catalog.
+
+. Uninstall the {asb-name} Operator.
+.. Navigate to *Catalog* -> *Operator Management* and select the
+*ansible-service-broker* project from the drop down.
```
Comment (ahardin-rh) [inline modules/sb-uninstall-asb.adoc]: s/drop down/drop-down menu

File: modules/sb-uninstall-tsb.adoc [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+* The {tsb-name} is installed.
+
+.Procedure
+
+This procedure assumes that you installed the {tsb-name} into the
+`template-service-broker` project.
+
+. Uninstall the {tsb-name}.
+.. Navigate to *Catalog* -> *Installed Operators* and select the
+*template-service-broker* project from the drop down.
```
Comment (ahardin-rh) [inline modules/sb-uninstall-tsb.adoc]: drop-down menu

File: modules/sb-uninstall-tsb.adoc [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+.. Click *Template Service Broker Operator*.
+.. Select the *Template Service Broker* tab.
+.. Click *template-service-broker*.
+.. From the *Actions* drop down, select *Delete Template Service Broker*.
```
Comment (ahardin-rh) [inline modules/sb-uninstall-tsb.adoc]: drop-down menu
confirmation page (or window?)

**Comment by ahardin-rh** [comment]
@bergerhoffer Just a few minor suggestions from me :+1:

**Comment by johnkim76** [comment]
@bergerhoffer 
LGTM!  Thank you!

**Pull request merged**